src/NHibernate.DomainModel/Northwind/Mappings/AnotherEntityRequired.hbm.xml

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<hibernate-mapping xmlns="urn:nhibernate-mapping-2.2" namespace="NHibernate.DomainModel.Northwind.Entities" assembly="NHibernate.DomainModel">
+  <class name="AnotherEntityRequired" table="AnotherEntity" mutable="false" schema-action="none">
+    <id name="Id">
+      <generator class="native" />
+    </id>
+    <property name="Output" not-null="true" />
+    <property name="Input" not-null="true" />
+    <property name="NullableOutput" formula="Output" lazy="true" />
```

Comment:
I've had to use lazy as otherwise an invalid query is generated due to the missing table alias for the column used in the formula.